node_modules/oae-core/manageaccess/js/manageaccess.js

```diff
@@ -29,6 +29,9 @@ define(['jquery', 'oae.core', 'jquery.autosuggest'], function($, oae) {
 
         // Caches the member updates that need to be applied
         var membersUpdates = {};
+        
+        // Caches members pending addition
+        var pendingMembers = {};
```

Comment:
I guess making it an array isn't entirely necessary since you want them to be unique. But reading through the difference of these isn't entirely clear. Maybe just needs more descriptive docs.